lib/makeup/lexers/elixir_lexer.ex

```diff
@@ -456,12 +456,12 @@ end
     case cond for if unless try receive raise
     quote unquote unquote_splicing throw super]
   @keyword_declaration ~W[
-    def defp defmodule defprotocol defmacro defmacrop
```

Comment:
The keywords `defguard` and `defguardp` need to be added to the `@def_like` list of keywords, so that they are highlighted just like `def(p)` and  `defmacro(p)`.

I admit I've merged this PR without paying much attention, sorry.
I'll fix it tomorrow.

Reply:
Maybe we should do `@keyword_declaration @def_like ++ [...]` to avoid duplication?